src/concerns/autoplay.coffee

```diff
@@ -0,0 +1,24 @@
+###
+Code related to auotplay features of the carousel
+###
+export default
+	props:
+		# A delay provided in seconds for the autoplay. 0 is disabled
+		autoplayDelay:
+			type: Number
+			default: 0
+
+		pauseOnFocus:
+			type: Boolean
+			default: true
+
+	computed:
+		# Stop animation if window is hidden or if carousel is focused
+		paused: -> (@windowHidden or @isFocused) if @pauseOnFocus
```

Comment:
I think you should increase the specificity of this name to `autoplayPaused`

I think you should make a watcher on this value that is like:

```coffee
watch:
  autoplayPaused: -> if @paused then @autoplayStop() else @autoplayStart()
```

Reply:
What's the purpose of the watcher on this? Is there something more to it being part of vue's data model rather than a computed property?

Reply:
With the current implementation, the interval never stops.  If you are hovering when the callback fires, it just skips advancing that time around.  If you stop hovering moments before the callback would fire, it will advance shortly after you mouseoff.

With my change, when you hover, we cancel the interval altogether.  So when you mouse off, there is a consistent delay before the next hover.

It's not a _big_ deal, but I think this approach is more intentional from a code and a ux place.

Reply:
Ahhh I see what the change is trying to fix